libs/challenge-registry/org-profile/src/lib/org-profile.component.ts

```diff
@@ -43,6 +53,24 @@ export class OrgProfileComponent implements OnInit {
   }
 
   ngOnInit(): void {
+    if (!this.useLocalOrganizationMock) {
+      this.organization$ = this.route.params.pipe(
+        switchMap((params) =>
+          this.organizationService.getOrganization(params['orgLogin'])
+        ),
+        catchError((err) => {
+          const error = err.error as ApiClientBasicError;
+          if (isApiClientError(error)) {
+            if (error.status === 404) {
+              // TODO: Redirect to org not found page or component
```

Comment:
is "org not found" different from the not-found page?

Reply:
We can redirect to the generic not-found page. I realize that I haven't see a "not found" page on the internet in a while, instead I'm believe I'm often silently redirected to the closest page or home page, which is probably more useful to start searching again rather than landing on the not found page.

What about we redirect to the home page when a resource is not found (separate ticket)?

Reply:
Interesting - I've had other experiences, where using an unknown search term would result in either a 404 page or "Term not found" page.  The "Not found" page would then either suggest using an advanced search or have a search bar to try again.

I've never experienced being redirected to the homepage before.

Reply:
Here is an example:

- Valid link: https://material.angular.io/components/button
- Invalid link ("button2"): https://material.angular.io/components/button2 => redirects to list of component: https://material.angular.io/components/categories

For an invalid organization link, we could redirect to the closest page (list of organizations?) + show show a [snack bar](https://material.angular.io/components/snack-bar/overview) with a message like "Organization not found".